zstd/VITABUILD

```diff
@@ -0,0 +1,16 @@
+pkgname=libzstd
+pkgver=1.4.4
+pkgrel=1
+url='https://zstd.net'
+source=("https://github.com/facebook/zstd/releases/download/v$pkgver/zstd-$pkgver.tar.gz")
+sha256sums=('59ef70ebb757ffe74a7b3fe9c305e2ba3350021a918d168a046c6300aeea9315')
```

Comment:
can you add dependency `boost` for zstd?

Reply:
why? it's the other way around, boost::iostreams depend on zstd

Reply:
ah sorry, i just misunderstand :)